.github/workflows/leaderboard.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Update Leaderboard
         id: leaderboard
         run: |
-          make update-leaderboard
+          make leaderboard
```

Comment:
Nice, but not related to update-all. If you want to squash all commits of this pr (since they are not planned), we sill have mix of many unrelated changes.

Reply:
just had to do that to Test if it works because it was running update-leaderboard First for some reason and faile don first

Reply:
Lets split this change so we can merge it quickly.

Reply:
it is kind of related to this PR, because update-all calls all the targets that start with word "update"
and if we dont do that, it will break the "update-all"